### PR TITLE
uri is now resolved if the env variable set in api. 

### PR DESCRIPTION
**Pull request by denisnikulin**
defaults to first element in uri to be a service which is suitable for docker

### CODE REVIEW

**Comment by Rabursky** [comment]
I don't think it solves anything. Actually it makes it less dynamic tbh...

File: packages/server/.env.dist [hunk 1]
```diff
@@ -1,2 +1,3 @@
 PORT=
-ACCOUNTS_URL=
\ No newline at end of file
+ACCOUNTS=http://0.0.0.0:3002
```
Comment (basia-horwath) [inline packages/server/.env.dist]: _ACCOUNTS_ & _TEAMS_ may be a bit misleading, 'cause they don't imply that these are micro-service addresses.

File: packages/server/src/graphql-server/resolvers.js [hunk 2]
```diff
@@ -1,18 +1,22 @@
 const GraphQLJSON = require('graphql-type-json');
 const httpClient = require('superagent');
 
-const resolveQuery = (uri) => {
+const resolveQuery = (resolvedUri) => {
```
Comment (basia-horwath) [inline packages/server/src/graphql-server/resolvers.js]: There's no need to complicate this 😉 The function only cares that the argument is a valid address. We can even just change it to _url_.

File: packages/server/src/graphql-server/resolvers.js [hunk 3]
```diff
@@ -1,18 +1,22 @@
...
-    query: (_, {uri, args}) => {
-      return resolveQuery(uri);
+    query: (_, { uri, args }) => {
+      // parse uri to get service location identifier
+      const service = uri.split('/', 1)[0];
+      // check if according variable set and hit the endpoint
+      const resolvedUri = uri.replace(service, process.env[service.toUpperCase()] || service);
+      return resolveQuery(resolvedUri);
     }
   },
```
Comment (basia-horwath) [inline packages/server/src/graphql-server/resolvers.js]: Now, an ancient programming secret is going to be revealed to you: 

    _One with whom the Force is, shall not have any need for comments that describe what their code is doing._

**Comment by basia-horwath** [comment]
It's working 🎉 

CI got stuck in the meantime, so just feel free to merge, when it finishes. I need to run, 'cause I'm late.

**Pull request merged**